src/Geta.NotFoundHandler.Admin/Areas/GetaNotFoundHandlerAdmin/Pages/Components/SortableHeaderCell/Default.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@using Geta.NotFoundHandler.Admin.Areas.GetaNotFoundHandlerAdmin.Pages.Models
+@model Geta.NotFoundHandler.Admin.Areas.GetaNotFoundHandlerAdmin.Pages.Components.SortableHeaderCell.SortableHeaderCellViewModel
+
+<a asp-route-sortColumn="@Model.Key"
+   asp-route-sortDirection="@(Model.SortDirection == SortDirection.Ascending ? SortDirection.Descending : SortDirection.Ascending)"
```

Comment:
Should it be possible to toggle no-sort as well?

Reply:
Added no-sort state.